### ISSUE TITLE
draksnapshot-config crashed (Bugzilla Bug 2280)

### ISSUE THREAD

**Issue by tpgxyz** [OpenMandrivaAssociation/test2#2280]
This issue was created automatically with bugzilla2github
# Bugzilla Bug 2280
Date: 2017-12-24 22:46:55 +0000
From: Geraldo Lopes &lt;<geraldo4fun@yahoo.com.br>&gt;
To:   @berolinux
CC:   @benbullard79, bugs@openmandriva.org, geraldo4fun@yahoo.com.br, @itchka, nix.or.die@gmail.com, nobodydead@gmail.com, @filjob, om-security@ml.openmandriva.org, @rugyada, @tpgxyz

Last updated: 2019-01-05 05:25:03 +0000

## Comment 17722
Date: 2017-12-24 22:46:55 +0000
From: Geraldo Lopes &lt;<geraldo4fun@yahoo.com.br>&gt;

The "draksnapshot-config" program crashed. Drakbug-16.73 caught it.

Ocorreu durante o processo de atualização do sistema.

you must be root to run this program at /usr/lib/libDrakX/common.pm line 592.
Perl's trace:
drakbug::bug_handler() called from /usr/lib/libDrakX/common.pm:592
common::require_root_capability() called from /usr/sbin/draksnapshot-config:41

Theme name: Breeze
Kernel version = 4.13.12-desktop-2omv
Distribution=OpenMandriva Lx release 3.0 (Einsteinium) for x86_64
CPU=Intel(R) Core(TM) i5-4440 CPU @ 3.10GHz

## Comment 17724
Date: 2017-12-24 23:52:03 +0000
From: @benbullard79

First question is what are you trying to do? 

Second question is what were you doing when program seemed to crash? 

Third question is did you see the part where drakbug tells you: "you must be root to run this program at /usr/lib/libDrakX/common.pm line 592"? 

Fourth question is have you tried to run 'draksnapshot-config' from commandline as root?

## Comment 17725
Date: 2017-12-25 18:58:53 +0000
From: @rugyada

(In reply to Ben Bullard from comment #1)
> 
> Fourth question is have you tried to run 'draksnapshot-config' from
> commandline as root?

There were some reports about "Snapshots" crashes when launched from OMCC (OpenMandriva Control Center), while it seems to work well from command line (draksnapshot-config in root console).

## Comment 17727
Date: 2017-12-26 21:45:25 +0000
From: @benbullard79

*** Bug #2281 has been marked as a duplicate of this bug. ***

## Comment 17728
Date: 2017-12-26 21:47:11 +0000
From: @benbullard79

Geraldo Lopes would you please answer the questions in Comment # 1?

## Comment 18842
Date: 2018-12-27 16:39:23 +0000
From: @filjob

making this issue mine :) and answering the questions:
1) I am tryinga backup solution for /home
2) in control center I clicked the draksnapshot icon and it crashed
3) no
4) yes, sudo draksnapshot-config
it starts, a window pops up and when I try to add /home to the list it crashed.

Below a list of warnings displayed in the console

Subroutine Gtk3::main redefined at /usr/lib/perl5/vendor_perl/5.20.3/Gtk3.pm line 525.

(draksnapshot-config:8377): Gtk-WARNING **: Theme parsing error: gtk.css:68:35: The style property GtkButton:child-displacement-x is deprecated and shouldn't be used anymore. It will be removed in a future version

(draksnapshot-config:8377): Gtk-WARNING **: Theme parsing error: gtk.css:69:35: The style property GtkButton:child-displacement-y is deprecated and shouldn't be used anymore. It will be removed in a future version

(draksnapshot-config:8377): Gtk-WARNING **: Theme parsing error: gtk.css:70:34: The style property GtkCheckButton:indicator-size is deprecated and shouldn't be used anymore. It will be removed in a future version

(draksnapshot-config:8377): Gtk-WARNING **: Theme parsing error: gtk.css:71:36: The style property GtkCheckMenuItem:indicator-size is deprecated and shouldn't be used anymore. It will be removed in a future version

(draksnapshot-config:8377): Gtk-WARNING **: Theme parsing error: gtk.css:73:46: The style property GtkScrolledWindow:scrollbars-within-bevel is deprecated and shouldn't be used anymore. It will be removed in a future version

(draksnapshot-config:8377): Gtk-WARNING **: Theme parsing error: gtk.css:76:30: The style property GtkExpander:expander-size is deprecated and shouldn't be used anymore. It will be removed in a future version

(draksnapshot-config:8377): Gtk-WARNING **: Theme parsing error: gtk.css:83:29: The style property GtkStatusbar:shadow-type is deprecated and shouldn't be used anymore. It will be removed in a future version

(draksnapshot-config:8377): Gtk-WARNING **: GtkImage 0x389a590 adjusted size vertical min 59 natural 59 must not decrease below min 64 natural 64

(draksnapshot-config:8377): Gtk-WARNING **: GtkImage 0x389a590 attempted to adjust its size allocation from -12,5 540x64 to 0,0 528x69. adjust_size_allocation must keep allocation inside original bounds
Subroutine Gtk3::main redefined at /usr/lib/perl5/vendor_perl/5.20.3/Gtk3.pm line 525.

(drakbug:8377): Gtk-WARNING **: Theme parsing error: gtk.css:68:35: The style property GtkButton:child-displacement-x is deprecated and shouldn't be used anymore. It will be removed in a future version

(drakbug:8377): Gtk-WARNING **: Theme parsing error: gtk.css:69:35: The style property GtkButton:child-displacement-y is deprecated and shouldn't be used anymore. It will be removed in a future version

(drakbug:8377): Gtk-WARNING **: Theme parsing error: gtk.css:70:34: The style property GtkCheckButton:indicator-size is deprecated and shouldn't be used anymore. It will be removed in a future version

(drakbug:8377): Gtk-WARNING **: Theme parsing error: gtk.css:71:36: The style property GtkCheckMenuItem:indicator-size is deprecated and shouldn't be used anymore. It will be removed in a future version

(drakbug:8377): Gtk-WARNING **: Theme parsing error: gtk.css:73:46: The style property GtkScrolledWindow:scrollbars-within-bevel is deprecated and shouldn't be used anymore. It will be removed in a future version

(drakbug:8377): Gtk-WARNING **: Theme parsing error: gtk.css:76:30: The style property GtkExpander:expander-size is deprecated and shouldn't be used anymore. It will be removed in a future version

(drakbug:8377): Gtk-WARNING **: Theme parsing error: gtk.css:83:29: The style property GtkStatusbar:shadow-type is deprecated and shouldn't be used anymore. It will be removed in a future version

## Comment 18843
Date: 2018-12-27 17:00:52 +0000
From: @filjob

more info: when trying draksnapshot from CC the error is:

  Undefined subroutine &main::create_factory_menu called at /usr/sbin/draksnapshot-config line 105.
  Perl's trace:
  drakbug::bug_handler() called from /usr/sbin/draksnapshot-config:105

I am running the LXQT distribution

## Comment 18844
Date: 2018-12-27 17:09:57 +0000
From: @benbullard79

This from OM Lx 4.0 Alpha1 Release Notes:

In order to keep current with latest changes in Linux, computer security issues, and computer code writing there are major changes in OM Lx4. ... In a major step forward OMCC/drakconf and drakxtools will be replaced with newer, more up to date, software. 

OK, back to bug report.

In reality it will take some time and hard work before everything has a replacement. Also due to the mentioned changes in Linux some things OM users got used to over the years may not be possible to duplicate. I do think some kind of snapshot utility is possible and probably already exists elsewhere.

What would be majorly helpful is to make a Feature request for software with same or similar function for Cooker. Developers are in process of developing replacement software and really need to hear from users what users want. 

One helpful thing users can do is to do internet search and see if you can find software that does this and add that to Feature request bug report.

To make a Package/Feature request bug report under Severity select 'feature'.

Reluctantly closing as RESOLVED/CANTFIX.

## Comment 18850
Date: 2018-12-30 03:37:18 +0000
From: @benbullard79

re open ing.

## Comment 18851
Date: 2018-12-30 03:41:55 +0000
From: @benbullard79

This probably should be a new bug report but for now changing this to a feature request for a replacement package for draksnapshot for Cooker so if this can be done it is in OM Lx 4.0 release.

For Geraldo and Olivier this is my best guess at the best way to handle this at this time. We shall see what devs say.

## Comment 18853
Date: 2018-12-30 03:58:46 +0000
From: @benbullard79

Apologies for confusion on this. I decided to file a new bug report myself. 

Replacement for draksnapshot. (Package request.)
https://issues.openmandriva.org/show_bug.cgi?id=2399

I did want OP Geraldo and Olivier to realize this was not ignored. 

Could use some suggestions for replacement software for the new bug report.

## Comment 18910
Date: 2019-01-05 05:14:33 +0000
From: @benbullard79

Hello, If anyone is still paying attention to this issue here's some suggestions.

They all are in OM repositories for Lx 3 and will be for Lx 4. The choices are:

KBackup
luckuBackup
Backupninja

All three will install in Lx 3 and Lx 4. KBackup and luckyBackup both open no problem. Not sure about backupninja.

So I'm hoping I can get Y'all to try KBackup and luckBackup and report on what you thing about each.

## Comment 18911
Date: 2019-01-05 05:25:03 +0000
From: @benbullard79

I made a mistake I should leave this as RESOLVED/CANTFIX. This is because developers are no longer able to maintain drakxtools. 

This from LX 4.0 Alpha1 Release Notes is a fancy way of saying why:

In order to keep current with latest changes in Linux, computer security issues, and computer code writing there are major changes in OM Lx4. 

Basically that means the drakxtools and some other code is to old and out of date to be maintainable on modern Linux systems.
